style.css

```diff
@@ -13,16 +13,87 @@ ul {
   margin: 0;
 }
 
-.body-m1 {
-  width: 400px;
-  background-color: #f4f5f7;
+.hamburger {
+  display: block;
+}
+
+.desktop-menu {
+  display: none;
 }
 
 .container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 36px 0 24px;
+  background: #fff;
+  height: 48px;
+  position: fixed;
+  width: 100vw;
+}
+
+/* .body-m1 {
+  width: 400px;
+  background-color: #f4f5f7;
+} */
+
+/* .container {
   display: flex;
   justify-content: space-between;
   align-items: center;
   background-color: white;
+} */
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  background-color: #6070ff;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+}
+
+.hamburger.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger.active .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+}
+
+.hamburger.active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+}
+
+.nav-menu {
```

Comment:
- [ ] Your colors seem to be inverted, please fix them so the menu looks like the Figma template.  The background should be #6070FF (blue) and the text and close button should be #FFFFFF (white).

Look:

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/157392345-ff8214e3-13fd-4881-ba2a-a1b3b6eb70ef.png) | ![image](https://user-images.githubusercontent.com/57421823/157392376-51fd61a2-9420-42ce-a7c3-2418c6920202.png)

According to the project requirements: 

![image](https://user-images.githubusercontent.com/57421823/157392590-22416440-9b0c-4a87-9d33-5a7995b09f68.png)

Therefore:

- [ ] Please, make it so when the user clicks or taps on any menu item, the correct part of the page is displayed.

If you implement these changes, your menu will look like the Figma, and it will work as intended! 💪